bindings/python/package/pyproject.toml

```diff
@@ -1,17 +0,0 @@
-[tool.poetry]
```

Comment:
If `poetry` is marked as a dependency somewhere in the project, that can be removed as well.